cpp/BUILD.bazel

```diff
@@ -60,13 +73,29 @@ genrule(
     srcs = [
         "default_worker",
         "ray_api",
+        "libray_api.so",
     ],
     outs = ["ray_cpp_pkg.out"],
     cmd = """
         WORK_DIR="$$(pwd)" &&
         mkdir -p "$$WORK_DIR/python/ray/core/src/ray/cpp/" &&
         cp -f $(location default_worker) "$$WORK_DIR/python/ray/core/src/ray/cpp/default_worker" &&
         cp -f $(locations ray_api) "$$WORK_DIR/python/ray/core/src/ray/cpp/" &&
+        cp -f $(locations libray_api.so) "$$WORK_DIR/python/ray/core/src/ray/cpp/" &&
+        mkdir -p "$$WORK_DIR/cpp/example/thirdparty/include/" &&
```

Comment:
Maybe we should copy files in `cpp/example/BUILD.bazel`? `ray_cpp_pkg ` don't know the example program.